docs/shared/monorepo-affected.md

```diff
@@ -54,6 +54,8 @@ Run `nx dep-graph` to see the dependency graph.
 
 ## Affected
 
+Affected projects are projects that are impacted by a set of changes. In order to find out which projects could be impacted by a particular change, Nx first determines which projects own the changed files. These projects are definitely impacted because they had changes directly to them. Projects that consume projects which are directly changed may also be impacted by these changes. As a result, those projects must be tested as well to verify that all changes in behavior are identified.
```

Comment:
These projects are definitely impacted because they had changes made directly to them.